examples/07_advanced/plot_advanced_decoding_scikit.py

```diff
@@ -85,9 +85,10 @@
 # the function :func:`sklearn.model_selection.cross_val_score` that computes
 # for you the score for the different folds of cross-validation.
 from sklearn.model_selection import cross_val_score
+# Here `cv=5` stipulates a 5-fold cross-validation
 cv_scores = cross_val_score(svc, fmri_masked, conditions, cv=5)
+print("Scikit-learn accuracy: {:.3f}".format(cv_scores.mean()))
```

Comment:
Maybe rather: SVC accuracy ?

Reply:
True. But we need to distinguish between these two prints::
* https://github.com/DimitriPapadopoulos/nilearn/blob/d99c6d8/examples/07_advanced/plot_advanced_decoding_scikit.py#L90
* https://github.com/DimitriPapadopoulos/nilearn/blob/d99c6d8/examples/07_advanced/plot_advanced_decoding_scikit.py#L109

Perhaps:
```python
print("SVC accuracy: {:.3f}".format(cv_scores.mean()))
```
```python
print("SVC accuracy (tuned parameters): {:.3f}".format(cv_scores.mean()))
```

Reply:
Done.